modules/efk-logging-fluentd-alerts.adoc

```diff
@@ -3,15 +3,32 @@
 // * logging/efk-logging-fluentd.adoc
 
 [id="efk-logging-fluentd-log-viewing_{context}"]
-= Viewing Fluentd logs
+= Viewing collected logs
 
-How you view logs depends upon the `LOGGING_FILE_PATH` setting.
+How you view logs collected by the log collector, Fluentd or Rsyslog, depends upon the `LOGGING_FILE_PATH` setting.
```

Comment:
These logs are not collected by the log collector, they are generated by the log collector.